### PR TITLE
Rover: remove duplicate get stopping location code

### PR DESCRIPTION
**Pull request by IamPete1**
while poking about in rover I noticed this functionality is duplicated in mode.cpp and AR_WPNav.cpp. This removes the mode.ccp version in favor of the AR_WPNav function that is now public.

should be no functional change, I have tested loiter still works in SITL.

### CODE REVIEW

**Comment by IamPete1** [comment]
the existing AR_WPNav function we now use can be found here https://github.com/ArduPilot/ardupilot/blob/2b0773c18b4d83636853c0a1dfa431ea77c4953c/libraries/AR_WPNav/AR_WPNav.cpp#L232

File: libraries/AR_WPNav/AR_WPNav.h [hunk 1]
```diff
@@ -114,8 +114,6 @@ class AR_WPNav {
     // adjust speed to ensure it does not fall below value held in SPEED_MIN
     void apply_speed_min(float &desired_speed);
 
-private:
```
Comment (rmackay9) [inline libraries/AR_WPNav/AR_WPNav.h]: no need to remove this "private" I think.

**Comment by rmackay9** [comment]
Looks good (apart from the "private" deletion).  I think this duplication appeared when I made the AR_WPNav library and I wasn't sure where to put this stopping_location() call.  It doesn't belong in attitude control because attitude control doesn't know about positions.  I felt weird about putting it in AR_WPNav because some modes use this call but don't use wpnav.

Anyway, better to have just one version so let's merge this after the one fixup is done.

File: libraries/AR_WPNav/AR_WPNav.h (hunk 1 above)
Comment (IamPete1) [reply libraries/AR_WPNav/AR_WPNav.h]: fixed

**Comment by IamPete1** [comment]
@rmackay9 thanks!,  I have put the private back ;)

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**